bugwarrior/services/redmine.py

```diff
@@ -220,7 +220,8 @@ def get_issue_url(self):
         return self.config.url + "/issues/" + str(self.record["id"])
 
     def get_converted_hours(self, estimated_hours):
-        tw = TaskWarriorShellout()
+        cfg = config.load_config('general', False, False)
+        tw = TaskWarriorShellout(config_filename=cfg['general'].taskrc)
```

Comment:
The configuration should already be loaded and available in `self.main_config`:
```suggestion
        tw = TaskWarriorShellout(config_filename=self.main_config.taskrc)
```

Reply:
Thank you! In the end, I just adopted both!...